data_model/src/query.rs

```diff
@@ -1249,6 +1254,35 @@ pub mod transaction {
         account::prelude::AccountId, expression::EvaluatesTo, transaction::TransactionValue,
     };
 
+    #[derive(
+        Default,
+        Debug,
+        Clone,
+        Copy,
+        PartialEq,
+        Eq,
+        PartialOrd,
+        Ord,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+    )]
+    /// `FindAllTransactions` Iroha Query will list all transactions included in blockchain
+    pub struct FindAllTransactions;
```

Comment:
I think we should support n last or from hash filtering in the `FindAllTransactions` query

Reply:
I don't know much about the use-cases, so I'll defer to your judgement here.

Reply:
`from_hash` — good idea. `n_last` could be emulated with pagination, so I'd add a new ticket to the backlog.

Reply:
yes, should be in a separate ticket. I'll resolve this then